misc/contrib/docker/Makefile

```diff
@@ -0,0 +1,31 @@
+MAINTAINER=nkeyapps
+TAG=nsupdate.info
+VER=$(shell git describe)
+UWSGI_UID=700
+UWSGI_GID=700
+BASE_DIR='../../..'
+DOCKER_DIR='misc/contrib/docker'
+
+all: prod
+
+prod:
+	cd $(BASE_DIR); docker build --build-arg BUILD=prod --build-arg uwsgi_uid=$(UWSGI_UID) --build-arg uwsgi_gid=$(UWSGI_GID) --build-arg DOCKER_DIR=$(DOCKER_DIR) -t $(MAINTAINER)/$(TAG):$(VER) --rm -f $(DOCKER_DIR)/build/Dockerfile .
+	docker tag $(MAINTAINER)/$(TAG):$(VER) $(MAINTAINER)/$(TAG):prod
+
+release: prod release_latest
+	docker push $(MAINTAINER)/$(TAG):$(VER)
+	@echo "*** Don't forget to create a tag by creating an official GitHub release."
```

Comment:
usually tagging happens before a release is made from that tag.

Reply:
Not sure I follow the implication. Let me know if I misunderstood your point, but this is the process to release a docker image, which will be run after a git tag is made in the nsupdate.info project on the master branch. So the docker release manager runs from a checkout of that tag something like: `git checkout 0.12.0 && make release`. This will build the image and tag it accordingly, then upload it to the docker registry.

Reply:
Oh ok, I see the comment is misleading. This is an artifact of the previous makefile. I'll clean it up.

Reply:
I have added an explicit requirement for a git tag when invoking the release target.